go.mod

```diff
@@ -248,3 +262,7 @@ replace (
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/vedhavyas/go-subkey => github.com/strangelove-ventures/go-subkey v1.0.7
 )
+
+replace github.com/cosmos/ibc-go/v7 => github.com/notional-labs/ibc-go/v7 v7.0.0-20230725115016-ed60bbb0ec6f
```

Comment:
I was told by IBC-Go team that here is an alpha version that uses sdk 50:
https://github.com/cosmos/ibc-go/tree/upgrade-sdk-v0.50-alpha1

We should use this instead.

Reply:
we should not.  

the alpha version there is not up to date with the sdk, and there have been breaking changes between the two.  

I agree that it is not good practice to have a replace statement like this one, so I'll see about getting a new release cut asap from this:

* https://github.com/cosmos/ibc-go/pull/4392

I'll come back tomorrow, I do reckon that we can get the release cut in the next 24h.